src/dbnode/namespace/convert.go

```diff
@@ -172,5 +203,31 @@ func OptionsToProto(opts Options) *nsproto.NamespaceOptions {
 			BlockSizeNanos: iopts.BlockSize().Nanoseconds(),
 		},
 		ColdWritesEnabled: opts.ColdWritesEnabled(),
+		RuntimeOptions:    toRuntimeOptions(opts.RuntimeOptions()),
+	}
+}
+
+// toRuntimeOptions returns the corresponding RuntimeOptions proto.
+func toRuntimeOptions(opts RuntimeOptions) *nsproto.NamespaceRuntimeOptions {
+	if opts == nil || opts.IsDefaults() {
+		return nil
+	}
+	var (
+		writeIndexingPerCPUConcurrency *protobuftypes.DoubleValue
+		flushIndexingPerCPUConcurrency *protobuftypes.DoubleValue
+	)
+	if v := opts.WriteIndexingPerCPUConcurrency(); v != nil {
+		writeIndexingPerCPUConcurrency = &protobuftypes.DoubleValue{
+			Value: *v,
+		}
+	}
+	if v := opts.FlushIndexingPerCPUConcurrency(); v != nil {
+		flushIndexingPerCPUConcurrency = &protobuftypes.DoubleValue{
+			Value: *v,
+		}
+	}
+	return &nsproto.NamespaceRuntimeOptions{
+		WriteIndexingPerCPUConcurrency: writeIndexingPerCPUConcurrency,
+		FlushIndexingPerCPUConcurrency: flushIndexingPerCPUConcurrency,
```

Comment:
If these are both set but to nil, `IsDefaults()` is true, right?

Reply:
Right yeah, so this code path doesn't return anything if IsDefaults() is true.